kura/distrib/config/kura.build.properties

```diff
@@ -13,7 +13,7 @@ org.eclipse.kura.deployment.agent.version=1.0.7
 org.eclipse.kura.core.deployment.version=1.1.2
 org.eclipse.kura.camel.version=1.1.0
 org.eclipse.kura.camel.cloud.factory.version=1.0.0
-org.eclipse.kura.camel.xml.version=1.0.0
+org.eclipse.kura.camel.xml.version=1.0.2-SNAPSHOT
```

Comment:
Why you need here to move and fetch 1.0.2-SNAPSHOT?
Since it was not distributed with 2.1.0, shouldn't be 1.0.1-SNAPSHOT?